packages/devtools/cli/src/commands/app/create.ts

```diff
@@ -125,16 +104,18 @@ export default class Create extends BaseCommand {
               pwa: true
             }
       });
-
-      await Promise.all(result.map((file) => file.save()));
-
-      this.log(`App created. To get started run the following commands:\n\n  cd ${name}\n  pnpm install\n  pnpm serve`);
+      await Promise.all(
+        result.map(async (file) => {
+          const saved = await file.save();
+          if (verbose) {
+            console.log(saved ? 'wrote' : 'skip', file.shortDescription());
+          }
+        })
+      );
     } catch (err: any) {
-      this.log(`Unable to create: ${err.message}`);
+      this.log('Unable to create application');
       this.log(err?.stack);
       this.error(err, { exit: 1 });
```

Comment:
```suggestion
      this.error(err);
      this.error(err?.stack);
      this.error('Unable to create application', { exit: 1 });
```
wondering if something like this might make more sense when reading the output?